kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExportHeaderGenerator.kt

```diff
@@ -812,11 +812,15 @@ internal class ObjCExportTranslatorImpl(
     private fun mustBeDocumentedAnnotations(annotations: Annotations): List<String> {
         return annotations.mapNotNull { it ->
             it.annotationClass?.let { annotationClass ->
-                if (!mustBeDocumentedAnnotationsStopList.contains(annotationClass.fqNameSafe) && annotationClass.annotations.any { metaAnnotation ->
-                            metaAnnotation.fqName == StandardNames.FqNames.mustBeDocumented
-                        })
-                    renderAnnotation(it, annotationClass)
-                else null
+                if (mustBeDocumentedAnnotationsStopList.contains(annotationClass.fqNameSafe)) return@let null
+                var mustBeDocumented = false
+                for (metaAnnotation in annotationClass.annotations) {
+                    when (metaAnnotation.fqName) {
+                        StandardNames.FqNames.mustBeDocumented -> mustBeDocumented = true
+                        KonanFqNames.refinesInSwift -> return@let null
```

Comment:
Hmm good point! It's not even that important since these declarations aren't "visible" in Swift anyway.
P.S. we don't need to add the ObjC variant since those aren't exported.

Reply:
Ah, right, I didn't even think about that!